doctrine.rst

```diff
@@ -347,9 +347,9 @@ and save it!
     class ProductController extends AbstractController
     {
         /**
-         * @Route("/product", name="product")
+         * @Route("/product", name="create_product"): Response
```

Comment:
The return type should be in the method, not the annotation I guess 😊 but hey... nice feature idea 😄